plugins/osdn/ovs/controller.go

```diff
@@ -294,6 +320,10 @@ func (c *FlowController) Setup(localSubnetCIDR, clusterNetworkCIDR, servicesNetw
 		return false, fmt.Errorf("Could not enable IPv4 forwarding on %s: %s", TUN, err)
 	}
 
+	// Table 253: rule version; note action is hex bytes separated by '.'
+	pluginVersion := getPluginVersion(c.multitenant)
+	otx.AddFlow("%s, %s%s.%s", VERSION_TABLE, VERSION_ACTION, pluginVersion[0], pluginVersion[1])
+
```

Comment:
I was going to say that it's unlikely that this would fail if everything before it succeeded, and even if it does, everything would still work fine, we'd just unnecessarily recreate everything next time...

But there is one way this could fail even if everything else succeeded, which is if we stupidly made a typo in this line and caused an ovs syntax error. But the "everything would still work fine" argument still holds. So maybe check the error and just log it if we get one?

Otherwise LGTM